backend/service/aws/dynamodb.go

```diff
@@ -118,8 +118,8 @@ func newProtoForIndexStatus(s types.IndexStatus) dynamodbv1.GlobalSecondaryIndex
 	return dynamodbv1.GlobalSecondaryIndex_Status(value)
 }
 
-func newProtoForBillingMode(s types.BillingMode) dynamodbv1.Table_BillingMode {
-	value, ok := dynamodbv1.Table_BillingMode_value[string(s)]
+func newProtoForBillingMode(s *types.BillingModeSummary) dynamodbv1.Table_BillingMode {
+	value, ok := dynamodbv1.Table_BillingMode_value[string(s.BillingMode)]
```

Comment:
Should this be a more explicit nil check?
something like:
```go
if s == nil {
		return dynamodbv1.Table_BILLING_UNKNOWN
}
value, ok := dynamodbv1.Table_BillingMode_value[string(s.BillingMode)]
```

Reply:
Good call, I'll add the explicit check